backend/de.metas.deliveryplanning.base/src/main/java/de/metas/deliveryplanning/DeliveryPlanningRepository.java

```diff
@@ -615,4 +614,12 @@ public Timestamp getMinLoadingDateFromCompletedDeliveryInstructions(@NonNull fin
 				.min(Timestamp::compareTo)
 				.orElse(null);
 	}
+
+	public void setPlannedLoadedQuantity(@NonNull final DeliveryPlanningId deliveryPlanningId, @NonNull final Quantity quantity)
+	{
+		final I_M_Delivery_Planning deliveryPlanning = getById(deliveryPlanningId);
+		deliveryPlanning.setPlannedLoadedQuantity(quantity.toBigDecimal());
```

Comment:
Please, also set the C_UOM_ID since it is also provided by Quantity.

Reply:
done.